spec/components/govspeak_spec.rb

```diff
@@ -12,6 +12,15 @@ def component_name
     assert_select ".gem-c-govspeak h1", text: "content"
   end
 
+  it "renders inverse content correctly" do
+    render_component(
+      inverse: true,
+      content: "<h2>inverse</h2>".html_safe,
```

Comment:
I believe that `raw()` is preferred over `.html_safe` - but I can't find the documentation to say so and why. They're effectively doing the same thing in this instance, so not a big deal.

```suggestion
      content: raw("<h2>inverse</h2>")
```

Reply:
This was based on the other tests in the file.